### PR TITLE
Service for Fop

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

**Comment by IegorSoboliev** [comment]
Ivan, please, look at the last commit for our coordination)

File: business_register/converter/fop.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from django.apps import apps
+
+class FopConverter(Converter):
+    bulk_manager = BulkCreateManager()
+
+    def save_to_db(self, records):
+        for record in records:
+            print(record.tag)
+            print(record.xpath('NAME')[0].text)
+            registration_text = record.xpath('REGISTRATION')[0].text
```
Comment (Anntroy) [inline business_register/converter/fop.py:14]: What does it mean .text?

File: business_register/converter/fop.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                estate_manager = record.xpath('ESTATE_MANAGER')[0].text,
+                termination_date = parse_date(self.get_first_word(termination_text)),
+                terminated_info = self.get_other_words(termination_text),
+                termination_cancel_info = record.xpath('TERMINATION_CANCEL_INFO')[0].text,
+                contact_info = record.xpath('CONTACTS')[0].text,
+                vp_dates = record.xpath('VP_DATES')[0].text,
+                authority = self.get_or_save_to_DB(record.xpath('CURRENT_AUTHORITY')[0].text, Authority)
+                )
+            self.bulk_manager.add(fop)
+        self.bulk_manager._commit(Fop)
```
Comment (Anntroy) [inline business_register/converter/fop.py:32]: What does it do _commit?

File: data_ocean/converter.py [hunk 3]
```diff
@@ -225,6 +207,36 @@ def process(self):
...
+        return string.split()[0]
+
+    def get_other_words(self, string):
+        return string.split()[1:]
+
+    def get_kved_from_DB(self, kved_code_from_record):
+        # verifying kved
+        empty_kved = Kved.objects.get(code='EMP')
+        kved_dict = {}    
+        for kved in Kved.objects.all():
```
Comment (Anntroy) [inline data_ocean/converter.py:221]: Please, verify about adding some value for objects.all()

File: data_ocean/converter.py [hunk 4]
```diff
@@ -225,6 +207,36 @@ def process(self):
 
     print('Converter has imported.')
 
+    #lets have all supporting functions in the end of the class
+    def get_first_word(self, string):
+        return string.split()[0]
+
+    def get_other_words(self, string):
```
Comment (criticus) [inline data_ocean/converter.py:214]: consider to rename it to make it more. clear

File: business_register/converter/fop.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
+from django.utils.dateparse import parse_date
+from data_ocean.converter import Converter, BulkCreateManager
+from business_register.models.rfop_models import Fop, FopToKved, ExchangeDataFop
+from data_ocean.models import Status, Authority, TaxpayerType
+from django.apps import apps
+
+class FopConverter(Converter):
+    bulk_manager = BulkCreateManager()
```
Comment (Iva-khar) [inline business_register/converter/fop.py:8]: You should to add constant CHUNK_SIZE

**Pull request closed without merging**